scripts/validation/service.schema.js

```diff
@@ -59,7 +59,7 @@ const schema = {
       properties: {
         fetch: {
           type: 'string',
-          pattern: '^https?://.+\.[pP][dD][fF]$',
+          pattern: '^https?:\/\/.+\.[pP][dD][fF]\??',
```

Comment:
> I don't understand why we need to escape`/`.

Also, I'd prefer that we keep validating that the input is on one line only: it is a very good sign that the contributor forgot to close the quote.

```suggestion
          pattern: '^https?://.+\.[pP][dD][fF](\\?.+)?$',
```

(tested with :

- https://investor.apple.com/files/doc_downloads/gov_docs/Apple-Human-Rights-Policy.pdf → passes
- https://investor.apple.com/files/doc_downloads/gov_docs/Apple-Human-Rights-Policy.pdf? → fails
- https://investor.apple.com/files/doc_downloads/gov_docs/Apple-Human-Rights-Policy.pdf?a → passes
)